src/test/java/store/mapper/EntityMapperTest.java

```diff
@@ -0,0 +1,62 @@
+package store.mapper;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import store.dto.ProductDTO;
+import store.model.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.eq;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EntityMapperTest {
+    @Mock
```

Comment:
this test class is not correct.
annotation mock is used to simulate behaviour of for example autowired beans.
dont use it to class which is testing